release/ml_user_tests/ray-lightning/ray_lightning_user_test.py

```diff
@@ -9,11 +9,18 @@
     start = time.time()
 
     addr = os.environ.get("RAY_ADDRESS")
-    job_name = os.environ.get("RAY_JOB_NAME", "horovod_user_test")
+    job_name = os.environ.get("RAY_JOB_NAME", "ray_lightning_user_test")
+
+    # Manually set NCCL_SOCKET_IFNAME to "ens3" so NCCL training works on
+    # anyscale_default_cloud.
+    # Passing in runtime_env to ray.init() will also set it for all the
+    # workers.
+    runtime_env = {"env_vars": {"NCCL_SOCKET_IFNAME": "ens3"}}
```

Comment:
does it make sense to instead change this to be exclusive (`^lo,^docker`)

Reply:
`docker` and `lo` are deprioritized by default. 

And we did try setting it to `^lo,docker` but that didn't work. 

For some reason, it only works if you explicitly set it to `ens3` or `ens5`. cc @matthewdeng

Reply:
interesting… maybe we should implement something like what Horovod implements (a nic checker)

Reply:
Yeah, can we check this in for now so that release tests are using NCCL, and create a follow-up ticket for a long-term solution? The problem currently seems to be with the `veth` interface specifically, but the exact cause is still under investigation.